src/pybind/mgr/influx/module.py

```diff
@@ -164,6 +169,10 @@ def get_daemon_stats(self):
             svc_type, svc_id = daemon.split(".")
             metadata = self.get_metadata(svc_type, svc_id)
 
+            # guard against OSD being removed while in this loop
+            if metadata == None:
```

Comment:
Prefer to use `is None` rather than `== None`.  It's not essential, but some editors highlight `== None` it as a style violation, so it's nicer to use the `is None`